src/main/java/com/iexec/worker/executor/TaskNotificationService.java

```diff
@@ -68,14 +74,16 @@ public TaskNotificationService(
             SubscriptionService subscriptionService,
             WorkerpoolAuthorizationService workerpoolAuthorizationService,
             IexecHubService iexecHubService,
-            SmsService smsService) {
+            SmsService smsService,
+            StompClientService stompClientService) {
         this.taskManagerService = taskManagerService;
         this.customCoreFeignClient = customCoreFeignClient;
         this.applicationEventPublisher = applicationEventPublisher;
         this.subscriptionService = subscriptionService;
         this.workerpoolAuthorizationService = workerpoolAuthorizationService;
         this.iexecHubService = iexecHubService;
         this.smsService = smsService;
+        this.stompClientService = stompClientService;
```

Comment:
I think we should use `SubscriptionService` instead overloading the constructor with a new service here.

Reply:
Good idea, please see https://github.com/iExecBlockchainComputing/iexec-worker/pull/497/commits/33b0892c7336fb67f8741c9eab60f9f496440f08.

Reply:
Thanks